packages/components/src/Table/Table.tsx

```diff
@@ -37,6 +37,7 @@ export interface TableProps
   variant?: string;
   size?: string;
   stretch?: boolean;
+  withStickyHeader?: boolean;
```

Comment:
shouldn't this be always sticky or am I get you wrong @sebald ? 🤔🙈

Reply:
I would make it optional, because it is actually a breaking change otherwise :D 

Can we also name it without the `with`, just `stickyHeader`?

Reply:
ah yes I see its in the issue :)